src/ubx.erl

```diff
@@ -115,6 +117,11 @@ poll_message(Pid, Msg, Payload) ->
     {MsgClass, MsgID} = resolve(Msg),
     gen_server:call(Pid, {poll_message, MsgClass, MsgID, Payload}).
 
+set_time_utc(Pid, DateTime) ->
+    MsgClass = ?MGA_INI,
+    MsgID = ?TIME_UTC,
+    gen_server:call(Pid, {send_message, MsgClass, MsgID, DateTime}).
```

Comment:
A better tag on this tuple would be wise. Eg `{set_time_utc, DataTime}`

Reply:
Agreed.  No reason to pass `MsgClass` and `MsgID` since these are constant for `set_time_utc`.

Reply:
You don't need to pass the msgclass and msgid if you have a descriptive tag on the message tuple.